File: Calculator program.java
import java.util.Scanner;
class calc
{
public static void main(String[] args)
{
int a, b,c;
char d;
System.out.println("Name: AVINASH");
System.out.println("REG.No: 2024503506");
System.out.println("Enter two numbers:");
Scanner x= new Scanner(System.in);
a = x.nextInt();
b = x.nextInt();
System.out.println("Enter the operation +, -, *, /:");
d=x.next().charAt(0);
if (d== '+')
{c=a+b;
System.out.println("a + b=" + c);}
else if (d== '-')
{c=a-b;
System.out.println("a-b =" + c);}
else if (d== '*')
{c=a*b;
System.out.println("ab="+ c);}
else if (d == '/')
{c=a/b;
System.out.println("a/b =" + c);}
else
System.out.println("wrong input");
}
}

File: Calculator program.cpp
#include <iostream>
using namespace std;

int main() {
    int a, b, c;
    char d;

    cout << "Name: AVINASH" << endl;
    cout << "REG.No: 2024503506" << endl;

    cout << "Enter two numbers:" << endl;
    cin >> a >> b;

    cout << "Enter the operation +, -, *, /: ";
    cin >> d;

    if (d == '+') {
        c = a + b;
        cout << "a + b = " << c << endl;
    } else if (d == '-') {
        c = a - b;
        cout << "a - b = " << c << endl;
    } else if (d == '*') {
        c = a * b;
        cout << "a * b = " << c << endl;
    } else if (d == '/') {
        if (b != 0) {
            c = a / b;
            cout << "a / b = " << c << endl;
        } else {
            cout << "Error: Division by zero" << endl;
        }
    } else {
        cout << "Wrong input" << endl;
    }

    return 0;
}
